### PR TITLE
Exponential learning rate

### PR DESCRIPTION
**Pull request by MarcoArmenta**

### CODE REVIEW

**Comment by MarcoArmenta** [comment]
> Nice review, although could you run your review through grammarly or the spellchecker of your choice ? There are lots and lots of typos in there. Also, could you give your PR a meaningful title ? Thanks

File: article/_posts/2020-05-28-ExponentialLR.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+cite:
+    authors: "Zhiyuan Li and Sanjeev Arora"
+    title:   "An exponential learning rate schedule for deep learning"
+    venue:   "ICLR 2019"
+pdf: "https://arxiv.org/pdf/1910.07454.pdf"
+---
+
+# Introduction
+
+In this paper they show that when using bathc normalization and weight decay one can use an **exponentially increasing** learning rate and still have good results. They also present a mathematical proof that the exponential learning rate is equivalent to BN + SGD + StandardRateTuning + WeightDecay + Momentum.
```
Comment (AntoineTheb) [inline article/_posts/2020-05-28-ExponentialLR.md:15]: bathc normalization

**Comment by MarcoArmenta** [comment]
Typos are not mine. They are from the paper, those are screen shots.

File: article/_posts/2020-05-28-ExponentialLR.md (hunk 1 above)
Comment (jodoin) [inline article/_posts/2020-05-28-ExponentialLR.md:15]: increasing or decreasing?

Comment (MarcoArmenta) [reply article/_posts/2020-05-28-ExponentialLR.md:15]: INCREASING

File: article/_posts/2020-05-28-ExponentialLR.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+* It is usual when using BN to lower the learning rate when the validation loss is plateau-ing.
+
+![](/article/images/ExponentialLR/sc01.jpg)   
+
+They use scale invariance to manage the computations on the proofs, namely:
+
+![](/article/images/ExponentialLR/sc02.jpg)   
+
+# Worm up
```
Comment (AntoineTheb) [inline article/_posts/2020-05-28-ExponentialLR.md:25]: Worm up ?

Comment (AntoineTheb) [inline article/_posts/2020-05-28-ExponentialLR.md:25]: Worm up ?

Comment (MarcoArmenta) [reply article/_posts/2020-05-28-ExponentialLR.md]: That was mine! Solved.

File: article/_posts/2020-05-28-ExponentialLR.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+cite:
+    authors: "Zhiyuan Li and Sanjeev Arora"
+    title:   "An exponential learning rate schedule for deep learning"
+    venue:   "ICLR 2019"
+pdf: "https://arxiv.org/pdf/1910.07454.pdf"
+---
+
+# Introduction
+
+In this paper they show that when using bathc normalization and weight decay one can use an **exponentially increasing** learning rate and still have good results. They also present a mathematical proof that the exponential learning rate is equivalent to BN + SGD + StandardRateTuning + WeightDecay + Momentum.
```
Comment (AntoineTheb) [inline article/_posts/2020-05-28-ExponentialLR.md:15]: bathc normalization

File: article/_posts/2020-05-28-ExponentialLR.md (hunk 1 above)
Comment (jodoin) [reply article/_posts/2020-05-28-ExponentialLR.md:15]: bathc , batch

File: article/_posts/2020-05-28-ExponentialLR.md (hunk 3 above)
Comment (MarcoArmenta) [reply article/_posts/2020-05-28-ExponentialLR.md]: Done

File: article/_posts/2020-05-28-ExponentialLR.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
+---
+layout: review
+title: "An exponential learning rate schedule for deep learning"
+tags: Exponential learning rate
```
Comment (jodoin) [inline article/_posts/2020-05-28-ExponentialLR.md:4]: This is not correct.  Please use the tags in _data/tags.yml

**Pull request merged**